KVMarketSquare/Favorites.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Favorites.swift
+//  KVMarketSquare
+//
+//  Created by Ka Hin Ng on 2021-12-13.
+//
+
+import Foundation
+
+class Favorites: ObservableObject {
+    // this should be of type some link to a shop identifier
+    @Published var shops: [String] = []
```

Comment:
yeah I think, it will be better if we have Store objects directly here. so we can easily use them directly. let me know how you guys think.

Reply:
That makes sense. `String` was a temporary type to try stuff.